bridgetown-core/lib/bridgetown-core/concerns/localizable.rb

```diff
@@ -13,12 +13,24 @@ def all_locales
                    end
 
       matching_resources = result_set.select do |item|
-        item.relative_path.parent == relative_path.parent && item.data.slug == data.slug
+        matches_resource?(item)
       end
 
       matching_resources.sort_by do |item|
         site.config.available_locales.index item.data.locale
       end
     end
+
+    def matches_resource?(item)
+      if item.relative_path.is_a?(String)
+        item.localeless_path == localeless_path
+      else
+        item.relative_path.parent == relative_path.parent
+      end && item.data.slug == data.slug
+    end
+
+    def localeless_path
+      relative_path.gsub("#{data.locale}/", "")
```

Comment:
This feels too broad — it would mangle any URLs with, e.g., `/en/sheep/oxen/pigs` suddenly becomes `/sheep/oxpigs`

Reply:
Good call. How about I do this.

```suggestion
      relative_path.gsub(/\A#{data.locale}\//, "")
```

Reply:
I'll go with this fornow.